src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -2,6 +2,19 @@ import React, { FunctionComponent } from "react";
 import { View, TextInput, StyleSheet } from "react-native";
 import { AppText } from "./AppText";
 import { size, color, borderRadius, fontSize } from "../../common/styles";
+import { PhoneNumberFormat, PhoneNumberUtil } from "google-libphonenumber";
+
+const singaporeNumberCode = "SG";
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+const phoneNumberFormatted = (mobileNumberValue: string): string => {
+  return mobileNumberValue.length >= 2
+    ? phoneUtil.format(
+        phoneUtil.parse(mobileNumberValue, singaporeNumberCode),
```

Comment:
This is another way of declaring default values, without having to instantiate a new `const` above, as well as allowing extensibility in the future if necessary:

```suggestion
const phoneNumberFormatted = (mobileNumberValue: string, region: string = "SG"): string => {
  return mobileNumberValue.length >= 2
    ? phoneUtil.format(
        phoneUtil.parse(mobileNumberValue, region),
```

Reply:
-Removed "SG" and used getRegionCodeForCountryCode so can fit formatting for International Numbers